### ISSUE TITLE
Output CSV : Uncaught TypeError: c.$table.off is not a function

### ISSUE THREAD

**Issue by NicholasNguyen06** [Mottie/tablesorter#1378]
Trying to output some tables to a CSV and used the basic JSFiddle example to replicate and im noticing an 
```
`Uncaught TypeError: c.$table.off is not a function
    at Object.init (widget-output.js:41)
    at Object.init (widget-output.js:422)
    at Object.applyWidgetId (jquery.tablesorter.js:1992)
    at Object.applyWidget (jquery.tablesorter.js:2055)
    at Object.setup (jquery.tablesorter.js:312)
    at HTMLTableElement.<anonymous> (jquery.tablesorter.js:2533)
    at Function.each (jquery-1.3.2.min.js:12)
    at init.each (jquery-1.3.2.min.js:12)
    at init.$.fn.tablesorter (jquery.tablesorter.js:2522)
    at HTMLDocument.<anonymous> (policyinfo:92)
init @ widget-output.js:41
init @ widget-output.js:422
applyWidgetId @ jquery.tablesorter.js:1992
applyWidget @ jquery.tablesorter.js:2055
setup @ jquery.tablesorter.js:312
(anonymous) @ jquery.tablesorter.js:2533
each @ jquery-1.3.2.min.js:12
each @ jquery-1.3.2.min.js:12
$.fn.tablesorter @ jquery.tablesorter.js:2522
(anonymous) @ policyinfo:92
(anonymous) @ jquery-1.3.2.min.js:19
each @ jquery-1.3.2.min.js:12
ready @ jquery-1.3.2.min.js:19
(anonymous) @ jquery-1.3.2.min.js:19`
```

Im not really sure where to start with it. Heres a gist of what of im doing. Thanks in advance for anyone that can take a look.

I've checked i have all the most recent updates,  updated all the files today, still no luck. Not really sure how to diagnose the error.

```
<script type="text/javascript" src="/js/jquery-1.3.2.min.js"></script>
<script type="text/javascript" src="/js/tablesorter/js/jquery.tablesorter.js"></script>
<script type="text/javascript" src="/js/tablesorter/js/jquery.tablesorter.widgets.js"></script>

<!-- Pager widget -->
<script type="text/javascript" src="/js/tablesorter/addons/pager/jquery.tablesorter.pager.js"></script>
<!-- output widget for table sorter -->
<script type="text/javascript" src="/js/tablesorter/js/widgets/widget-output.js"></script>
<script type="text/javascript" src="/js/tablesorter/js/parsers/parser-input-select.js"></script>

<script>
jQuery(document).ready(function($){
 var $table = ('#vehicleTable');
  $('.download').click(function(){
      $('#vehicleTable').trigger('outputTable');
      console.log(" clicked");
  });
  $("#vehicleTable").tablesorter({
    widgets: ['output']
  });
});
</style>
<button class="download">Get CSV</button>
<table class="tablesorter" id="vehicleTable">
<!-- heres the table stuff -->
</table>
```

**Comment by Mottie** [comment]
Hi @NicholasNguyen06!

jQuery's [`.off()`](http://api.jquery.com/off/) method is only available in v1.7+. If you look at the [output widget](https://mottie.github.io/tablesorter/docs/example-widget-output.html) documentation page under "requirements", you'll see that the widget requires jQuery v1.7+.

jQuery version 1.3.2 . was released in 2009 and it does have [security vulnerabilities](https://novasecure.neonova.net/threats/details.cgi?id=505475) so please make sure to use the latest version if at all possible.

**Comment by NicholasNguyen06** [comment]
@Mottie Thank you sir. Much appreciated. Fixed and working now.
